src/bp-forums/forums/template.php

```diff
@@ -892,6 +892,9 @@ function bbp_list_forums( $args = '' ) {
 				$counts = $r['count_before'] . implode( $r['count_sep'], $count ) . $r['count_after'];
 			}
 
+			// Change the style of link tag.
+			$r = apply_filters( 'bb_forum_link_custom_attr', $r, $title );
+
```

Comment:
@kishanjasani Can't you use any of these filter `bbp_before_list_forums__parse_args` & `bbp_after_list_forums__parse_args` to make it work?

Reply:
I don't think we need to add a new filter because we have already a filter to change the args.